### ISSUE TITLE
Scrape web for recipes and add them to database

### ISSUE THREAD

**Issue by leilamar** [nyu-software-engineering/recipe-recommender#55]

**Comment by dvataksi16** [comment]
Websites to scrape from?
allrecipes.com

**Comment by millyschiff** [comment]
- [ ] for all recipes, should be same format - if not we'll deal with it some how 
- [ ] add cuisine type tag to recipe scheme
- [ ] scrape recipes by cuisine type

**Comment by dvataksi16** [comment]
I agree, recipes should be categorized by a tag but scraped in categories!
